hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -348,11 +361,9 @@ sudo rpi-eeprom-update -a -d
 ```
 
 #### Changing the boot mode
-The easiest method to change the the default boot mode [raspi-config](../../configuration/raspi-config.md) to modify the `Boot Options`. Alternatively, edit the EEPROM configuration file to select the desired `BOOT_ORDER`.
+The easiest method to change the the default boot mode [raspi-config](../../configuration/raspi-config.md) to modify the `Boot Options`. Alternatively, edit the EEPROM configuration file to set the `BOOT_ORDER` according to the desired boot mode.
```

Comment:
This still says "The easiest method to change the the default boot mode raspi-config to modify..." which IMHO doesn't look right.